include/fc/static_variant.hpp

```diff
@@ -380,7 +501,7 @@ class static_variant {
     template<typename X>
     static constexpr int position() { return impl::position<X, Types...>::pos; }
 
-    template<int Pos, std::enable_if_t<Pos < impl::type_info<Types...>::size,int> = 1>
+    template<int Pos, std::enable_if_t<Pos < type_info::size,int> = 1>
     using type_at = typename impl::type_at<Pos, Types...>::type;
 };
 
```

Comment:
Are you planning to add some tests in the eos repo?

Reply:
If we want tests, I believe they should be in the `fc` repo.

Reply:
They *should* be there. But that means additional work to enable `fc` tests.

Reply:
we do have one fc test enabled